js/ui/dnd.js

```diff
@@ -183,11 +183,11 @@ var _Draggable = new Lang.Class({
         // didn't start the drag, to drop the draggable in case the drag was in progress, and
         // to complete the drag and ensure that whatever happens to be under the pointer does
         // not get triggered if the drag was cancelled with Esc.
-        if (event.type() == Clutter.EventType.BUTTON_RELEASE) {
+        if (event.type() === Clutter.EventType.BUTTON_RELEASE) {
             this._buttonDown = false;
             if (this._dragInProgress) {
                 return this._dragActorDropped(event);
-            } else if (this._dragActor != null && !this._animationInProgress) {
+            } else if (this._dragActor !== null && !this._animationInProgress) {
```

Comment:
Changes like this might have side effects, When you do non-strict comparison against `null`, we are sometimes intentionally checking for `undefined` as well.

Reply:
Yes. This PR is scary. I haven't gotten an environment yet to test this in (waiting for LM 19 BETA). Those issues need to be found and fixed. This PR might however fix other subtle bugs introduced by not using these strict comparisons.

Reply:
Try this http://mirrors.advancedhosters.com/linuxmint/isos/testing/linuxmint-19-cinnamon-64bit-beta.iso